src/components/pages/rules/rulesGrid/rulesGrid.js

```diff
@@ -37,13 +46,26 @@ export class RulesGrid extends Component {
     }
   };
 
+  openEditRuleFlyout = () => { this.setState({ openFlyoutName: 'edit' }); }
+
+  setSelectedRule = (rule) => { this.setState({ selectedRule: rule }); }
```

Comment:
Change `rule` to `selectedRule` so you can use `this.setState({ selectedRule });`
You don't need the `{}`, just use `selectedRule => this.setState({ selectedRule });`